tools/crm_resource_runtime.c

```diff
@@ -1797,7 +1789,7 @@ cli_resource_execute_from_params(pcmk__output_t *out, const char *rsc_name,
             for (lpc = 0; lpc < more; lpc++) {
                 if (local_copy[lpc] == '\n' || local_copy[lpc] == 0) {
                     local_copy[lpc] = 0;
-                    printf(" >  stdout: %s\n", local_copy + last);
+                    out->info(out, " >  stdout: %s", local_copy + last);
```

Comment:
I think this would be useful in XML, like ->subprocess_output(). Not sure if that could just be used directly

Reply:
Sure, subprocess_output just takes strings.  There's no reason we couldn't use that here.